tests/test_entities.py

```diff
@@ -169,19 +169,19 @@ def test_pass(self):
 
 def elements_equal(e1, e2):
     if e1.tag != e2.tag:
-        print('Tag: %s != %s' % (e1.tag, e2.tag))
+        print(('Tag: %s != %s' % (e1.tag, e2.tag)))
```

Comment:
I don't know why it thinks it needs this :thinking:

Seems to work the same way in py2 and py3:

```
$ python3 -c "print ('tag %s %s' % ('a', 'b'))"                       
tag a b
$ python2 -c "print ('tag %s %s' % ('a', 'b'))"                       
tag a b
```

Reply:
Hm, strange ... but I leave it as is if it's ok?